src/components/Settings/GroupPopup.js

```diff
@@ -72,14 +72,14 @@ export default function GroupPopup({ open, close, updateBubbles }) {
             padding="0px"
             className={classes.tabs}
           >
-            <Tab label="Gruppe erstellen" {...a11yProps(0)} width="100%" style={{ padding: 0 }} />
             <Tab label="Gruppe beitreten" {...a11yProps(1)} width="100%" />
+            <Tab label="Gruppe erstellen" {...a11yProps(0)} width="100%" style={{ padding: 0 }} />
```

Comment:
```suggestion
            <Tab label="Gruppe beitreten" {...a11yProps(0)} width="100%" />
            <Tab label="Gruppe erstellen" {...a11yProps(1)} width="100%" style={{ padding: 0 }} />
```

Und was ist mit dem Padding? Soll der auch dem anderen Tab hinzugefügt werden?